frontend/src/NetworkContext.js

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+
+const NetworkContext = createContext();
+
```

Comment:
I know I've been creating/using a lot of contexts. I swear I'm not context happy, there just wasn't an easy path on how to pass down "the backend failed" through the navigator.